README.md

```diff
@@ -1,2 +1,23 @@
 # Create database users
-Once the consignment api database is created, we need to create users for the API and the migration task to use. This means that we don't have to use the master username and password to access the database, we can control permissions using GRANT and we can use IAM authentication for the database. 
\ No newline at end of file
+Once the consignment api database is created, we need to create users for the API and the migration task to use. This means that we don't have to use the master username and password to access the database, we can control permissions using GRANT and we can use IAM authentication for the database. 
+
+## Running the lambda in AWS
+As this is something that we only need to run once after the database is created, we just need to run the lambda manually.
+
+`aws lambda invoke --function-name tdr-create-db-users-$STAGE`
+
+## Running the lambda locally.
+If you want to run this against a local database, there are a few extra steps.
+
+Create the database if it doesn't exist. It will need the correct username and password. You can use docker
+
+`docker run --name postgres -e POSTGRES_PASSWORD=password -e POSTGRES_USER=tdr -e POSTGRES_DB=consignmentapi -d -p 5432:5432 postgres`
```

Comment:
This is using the same container name as for the consignment db local docker. Is it worth changing this name for the testing instructions, to avoid clashes on local machines, ie to `... post-gres-db-users`

Reply:
Yeah good point. I only named it something simple because I drop and recreate it so often but I've changed this one.